src/Modules/UngradedModules/SubquoModule.jl

```diff
@@ -1841,10 +1841,12 @@ by submodule with 3 generators
 function quotient(M::SubquoModule, J::Ideal)
   @assert base_ring(M) == base_ring(J)
   M_quo = isdefined(M, :quo) ? M.quo : SubModuleOfFreeModule(ambient_free_module(M), Vector{elem_type(ambient_free_module(M))}())
-  U = M.sub+M.quo
+  U = M.sub+M_quo
```

Comment:
The changes look reasonable. But yet, I'm wondering: The modules have a field `.sum` which holds the exactly this expression. Shouldn't we use that or an appropriate getter function for it?